SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp

```diff
@@ -90,6 +90,34 @@ template<typename T> QWidget* make_integer_table_cell(QWidget& parent, T& value)
     return box;
 }
 
+
+QWidget* make_limited_integer_table_cell(QWidget& parent, int& value, int min, int max){

```

Comment:
Can we actually change this to either `int32_t` or `int64_t`. The size of `int` is not defined for C++.

Reply:
done!